### DISCUSSION TITLE
응답체계를 어떻게 할것인가

### DISCUSSION THREAD

**Discussion by happymink** [Strong-Potato/TripVote-BE#11] [Polls]
Controller에서 API에 대한 값을 응답할 때, 어떤 형식으로 응답을 보내는게 좋을까요?

**Comment by Dr-KoKo** [comment]
부가설명하자면, `Controller`에서 1. `ResponseEntity`를 반환할 지 2. `ApiResponse`를 반환할 지 에 관한 투표사항입니다.
